routes/index.js

```diff
@@ -0,0 +1,139 @@
+var express = require('express');
+var router = express.Router();
+require('console.html');
+//console.html('<body><h1>Hello, world!</h1></body>');
+
+/* GET home page. */
+router.get('/', function (req, res) {
+  res.render('index');
+});
+
+var food = ["chocolates", "chocolate"]
+var medicine = ["headache"];
+
+router.post('/', function (req, res) {
+  var problem = req.body.problem;
+  //res.write("Hello");
+  //console.log(typeof problem);
```

Comment:
remove comments like `console.log`, commented `res.write` and do commenting on the functionality flow